wrappers/nodejs/test/test-functional-online.js

```diff
@@ -46,3 +47,21 @@ describe('Disparity transform tests', function() {
     pipe.stop();
   });
 });
+
+describe('Points.exportToPly', function() {
+  it('exportToPly', () => {
+    const pc = new rs2.PointCloud();
+    const pipe = new rs2.Pipeline();
+    const file = 'points.ply';
+    pipe.start();
+    const frameset = pipe.waitForFrames();
+    const depth = frameset.depthFrame;
+    const points = pc.calculate(depth);
+    const color = frameset.colorFrame;
```

Comment:
Do not you think too much const but only used once?

Reply:
But it seems no difference if we change to use let.
We can save characters by using only one const like this:
```
const a = foo,
          b = bar;
```
but lint would warn that we should use multiple lines.

Reply:
Sorry, I misunderstood it, I will remove some unnecessary declarations.